src/ucp/rndv/rndv_am.c

```diff
@@ -28,15 +20,19 @@ static void ucp_rndv_am_probe_common(ucp_proto_multi_init_params_t *params)
         return;
     }
 
-    params->super.min_length = 0;
-    params->super.max_length = SIZE_MAX;
-    params->super.overhead   = context->config.ext.proto_overhead_multi;
-    params->super.latency    = 0;
-    params->first.lane_type  = UCP_LANE_TYPE_AM;
-    params->middle.lane_type = UCP_LANE_TYPE_AM_BW;
-    params->super.hdr_size   = sizeof(ucp_request_data_hdr_t);
-    params->max_lanes        = context->config.ext.max_rndv_lanes;
-    params->opt_align_offs   = UCP_PROTO_COMMON_OFFSET_INVALID;
+    params->super.cfg_thresh   = ucp_proto_rndv_cfg_thresh(
+                                         context, UCS_BIT(UCP_RNDV_MODE_AM)),
+    params->super.cfg_priority = 80;
```

Comment:
Yes, and that was the issue. If we set higher priority for am_zcopy it will disable all other rndv protocols with priority 80.

Reply:
you mean when config is set?

Reply:
yes, I do.

Reply:
I would rather say that previously it was important to have different priority for this protocols because we did change cfg_thresh basing on ZCOPY/BCOPY thresh for these protocols. So it was important because if `BCOPY_THRESH=100` and `ZCOPY_THRESH=1000` rndv/am/zcopy should have higher priority to be forced after 1000. But since now we have decided to care only about `rndv_mode` for setting `cfg_thresh`, there is no need to set different `cfg_priority` anymore